src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -69,6 +69,7 @@ module gsi_rfv3io_mod
 
   real(r_kind),dimension(:,:  ),allocatable:: ges_ps_bg 
   real(r_kind),dimension(:,:  ),allocatable:: ges_ps_inc 
+  real(r_kind),dimension(:,:  ),allocatable:: ges_ps_write
```

Comment:
ges_ps_write is used only in subroutine wrfv3_netcdf. Should it be defined inside that subroutine?

Reply:
Changed as suggested